src/Agent.Plugins/PipelineArtifact/Telemetry/PipelineArtifactActionRecord.cs

```diff
@@ -3,8 +3,8 @@
 
 using System;
 using Agent.Sdk;
+using Agent.Sdk.Blob;
```

Comment:
curious why we'd need this.  Doesn't seem like any other code has changed.

Reply:
It's needed to implement `PipelineTelemetryRecord`